test/jdk/jdk/internal/platform/cgroup/CgroupV1SubsystemControllerTest.java

```diff
@@ -64,15 +64,28 @@ public void testCgPathNonEmptyRoot() {
         assertEquals(expectedPath, ctrl.path());
     }
 
+    /*
+     * Less common cases: Containers
+     */
```

Comment:
Not really sure why this comment was added, is it refering to 'container mode'  mentioned in the comment above in another file?

Reply:
It's because there are already "Common case: Containers" and "Common case: Host". The old test testCgPathSubstring() and the new test testCgPathToMovedPath() do not belong to "Common case: Host" that comes just before them.